android/src/main/java/io/customer/reactnative/sdk/messagingpush/RNCIOPushMessaging.kt

```diff
@@ -38,6 +51,44 @@ class RNCIOPushMessaging(
         reactContext.addActivityEventListener(this)
     }
 
+    /**
+     * Adds push messaging module to native Android SDK based on the configuration provided by
+     * customer app.
+     *
+     * @param builder instance of CustomerIOBuilder to add push messaging module.
+     * @param config configuration provided by customer app for push messaging module.
+     */
+    internal fun addNativeModuleFromConfig(
+        builder: CustomerIOBuilder,
+        config: Map<String, Any>
+    ) {
+        val androidConfig = config.getTypedValue<Map<String, Any>>(key = "android") ?: emptyMap()
+        // Use android specific push configurations for only Android specific configurations
+        // For common configurations, use the default push module configurations
+
+        // Auto track push events is enabled by default
+        // Fallback to common config if not provided in android specific config as this is a common
+        // configuration available for both Android and iOS
+        val autoTrackPushEvents =
+            androidConfig.getTypedValue<Boolean>(Keys.Config.AUTO_TRACK_PUSH_EVENTS)
+                ?: config.getTypedValue<Boolean>(Keys.Config.AUTO_TRACK_PUSH_EVENTS)
```

Comment:
From tech plan, looks like we discussed only removing this for iOS because it is no longer configurable in SDK. That makes me think we should continue exposing this in Android? Open to ideas.

Also, I added fallback to default `push` config, but I'm okay simplifying this and expecting this in `push.android` only.

Reply:
This might get a problem for us to maintain, if we are only allowing it via `android` we should only check in this config as well.